app/code/Magento/Sitemap/Model/Observer.php

```diff
@@ -17,11 +17,6 @@ class Observer
      */
     const XML_PATH_GENERATION_ENABLED = 'sitemap/generate/enabled';
 
-    /**
-     * Cronjob expression configuration
-     */
-    const XML_PATH_CRON_EXPR = 'crontab/default/jobs/generate_sitemaps/schedule/cron_expr';
```

Comment:
I would suggest to deprecate this instead of removing

Reply:
I would agree, but a) Magento does not follow semantic versioning between marketing releases and b) it points at the wrong config path so it's incredibly unlikely this is used anywhere directly.

Reply:
In any way it is better to stay on a safe side and follow conventions where possible.